src/desktop/apps/user/components/auction_registrations/view.coffee

```diff
@@ -30,5 +31,5 @@ module.exports = class AuctionRegistrationsView extends Backbone.View
 
   render: ->
     @$el.html template
-      sale_registrations: @me.sale_registrations
+      sale_registrations: @me.sale_registrations?.edges.map (e) -> e.node
```

Comment:
Not too experienced with `CoffeeScript`. Is this a no-no?

Reply:
I think this is fine. Separately we can also consider a mobbing session to convert this to ts/js.